### ISSUE TITLE
Unable to reproduce results

### ISSUE THREAD

**Issue by Rong-Tao** [xavysp/DexiNed#133]
I am training the model on my own dataset, where The original image (subtract mean with what you provided)
![1](https://github.com/xavysp/DexiNed/assets/51011533/40e25b67-ca55-43cd-ab0e-ae4d21563324)
and ground truth:
![2](https://github.com/xavysp/DexiNed/assets/51011533/f1469fe7-58cd-4f92-9059-02508ba1ca65)
The dataset uses 1000 images with ground truth edges 
all cropped to 512*512, and even after 40 epochs, the results are still horrible.
Setting:
```
criterion = bdcn_loss2
model = DexiNed().to(device)
optimizer = optim.Adam(model.parameters(), lr = 1e-4,  weight_decay = 1e-8)
```
Training code:
```
l_weight = [0.7,0.7,1.1,1.1,0.3,0.3,1.3]
for epoch in range(num_epochs):
    model.train()  # Set model to training mode
    running_loss = 0.0

    for batch_id, (raw_data, edge_data) in enumerate(train_loader):
        images, labels = raw_data.to(device), (edge_data > 0).float().to(device)

        # Forward pass
        preds_list = model(images)

        # Calculate loss
        loss = sum([criterion(preds, labels, l_w) for preds, l_w in zip(preds_list, l_weight)])

        # Backward pass and optimize
        optimizer.zero_grad()
        loss.backward()
        optimizer.step()

```
The 7 channels are as the following
![epoch_40_pred_1](https://github.com/xavysp/DexiNed/assets/51011533/59b27ab3-2daa-4ab8-bf0e-650991fe2adf)
![epoch_40_pred_2](https://github.com/xavysp/DexiNed/assets/51011533/2ff6973d-f563-47b5-b9df-38a62b179238)
![epoch_40_pred_3](https://github.com/xavysp/DexiNed/assets/51011533/677769a8-682a-4fae-b7f0-2363c232ead7)
![epoch_40_pred_4](https://github.com/xavysp/DexiNed/assets/51011533/b7032268-1201-4a60-aaaf-3885e6b934e9)
![epoch_40_pred_5](https://github.com/xavysp/DexiNed/assets/51011533/a69774ec-7a43-403c-9cdd-506758de076c)
![epoch_40_pred_6](https://github.com/xavysp/DexiNed/assets/51011533/3d801260-d45b-4942-b121-0fa0c192f4a3)
![epoch_40_pred_7](https://github.com/xavysp/DexiNed/assets/51011533/3d6bcc50-6361-4e13-80a8-72bb41632cc1)

**Comment by xavysp** [comment]
Hi, according to the images the model is not training, it's strange. Your result looks like they are just in the first two epochs. When we train with BIPED, we augment the data 200 by 280. This mean that in one epoch you used 56000 images. If you just used 1000 images this means that your 40 epochs used just 40000 images.

**Comment by Rong-Tao** [comment]
Ok, my original Image is `1024*768`, I will do random cropping and run it for `56*20` epochs, which should be equivalent to your 20 epochs. I've uploaded the code to the server I will let you know once I get the results

**Comment by Rong-Tao** [comment]
This is my tensorboard result, I will just cancel the training since it plateaued for 2 hours right now(I do added a lr.scheduler) ![tsbd](https://github.com/xavysp/DexiNed/assets/51011533/62740059-37fe-4ea0-9215-4648dab3ef3b). And the provided images are very similar to the images provided last time

**Comment by xavysp** [comment]
> Ok, my original Image is `1024*768`, I will do random cropping and run it for `56*20` epochs, which should be equivalent to your 20 epochs. I've uploaded the code to the server I will let you know once I get the results

I think you need to use the augmentation methods of BIPED, check the BIPED repo and let me know.  Maybe we can have a chat by zoom, I'm interested  how ends your proyect.

**Comment by xavysp** [comment]
> This is my tensorboard result, I will just cancel the training since it plateaued for 2 hours right now(I do added a lr.scheduler) ![tsbd](https://private-user-images.githubusercontent.com/51011533/287197937-62740059-37fe-4ea0-9215-4648dab3ef3b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE0MjUwNTUsIm5iZiI6MTcwMTQyNDc1NSwicGF0aCI6Ii81MTAxMTUzMy8yODcxOTc5MzctNjI3NDAwNTktMzdmZS00ZWEwLTkyMTUtNDY0OGRhYjNlZjNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjAxVDA5NTkxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTczYWUzNzIwMzYwNTc0NWYyZmI0YTQyNGY4YTBlZWY5MTdlNmJmNTEwN2Y5ZWM4ZDczM2I0MjA3OWZjZjM5YTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.XoxTO570ncL7ZK8pkUriNBH5nEXcPm8FVVJFhFiqyDE). And the provided images are very similar to the images provided last time

Can you tell me which kind of  machine are you using for training?  With dexined 2 hours is a short time.

**Comment by Rong-Tao** [comment]
Hey, I am using GPU equivalent to Nvidia V100 with 16GB Vram but it is neither Nvidia nor Amd. I will check the augmentation some time. I would be happy to have a zoom call some day with you once the new computation result is out. I guess you are in Spain? so we can probably do around 18:00 Beijing Time, basically +7 for EU time zone.

**Comment by xavysp** [comment]
> Hey, I am using GPU equivalent to Nvidia V100 with 16GB Vram but it is neither Nvidia nor Amd. I will check the augmentation some time. I would be happy to have a zoom call some day with you once the new computation result is out. I guess you are in Spain? so we can probably do around 18:00 Beijing Time, basically +7 for EU time zone.

Ok, let me know when you get the result. Actually, I'm in Ecuador, You may write me by my email  in the github profile.

Have a good day.
